# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/moz.build
# Commit: 2cf5cad90099
# Full Hash: 2cf5cad900992236d596bfb329099742ff15f689
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2024-06-20 04:08:16
# Regressor Bug: 1728331
# File Overlap Count: 1
# Description:
#   Bug 1728331 - Part 4: Make ContentParent KeepAlives explicit with RAII references, r=smaug,dom-worker-reviewers,asuth
#   
#   This is a fairly significant patch, however it would be difficult to break it
#   down into smaller patches:
#   
# ==============================================================================

diff -r d920c2d72d00 -r 2cf5cad90099 dom/ipc/moz.build
--- a/dom/ipc/moz.build	Wed Jun 19 20:14:49 2024 +0000
+++ b/dom/ipc/moz.build	Wed Jun 19 20:14:50 2024 +0000
@@ -85,6 +85,7 @@
     "TabContext.h",
     "TabMessageTypes.h",
     "TabMessageUtils.h",
+    "UniqueContentParentKeepAlive.h",
     "URLClassifierChild.h",
     "URLClassifierParent.h",
     "UserActivationIPCUtils.h",
@@ -148,6 +149,7 @@
     "SharedStringMap.cpp",
     "StructuredCloneData.cpp",
     "TabContext.cpp",
+    "UniqueContentParentKeepAlive.cpp",
     "URLClassifierParent.cpp",
     "WindowGlobalActor.cpp",
     "WindowGlobalChild.cpp",